infrastructure/template.yaml

```diff
@@ -1853,19 +1853,16 @@ Resources:
       AlarmActions:
         - !ImportValue platform-alarm-warning-alert-topic
       InsufficientDataActions: []
-      AlarmDescription: !Sub "${CheckHmrcEventBus} PutEvents (PutEvents Failed) 4 or more requests in the last hour. ${SupportManualURL}"
+      AlarmDescription: !Sub "${AWS::StackName}-${Environment} PutEvents (PutEvents Failed) 4 or more requests in the last hour. ${SupportManualURL}"
       AlarmName: !Sub "${AWS::StackName}-${Environment}-CheckHmrcEventBus-PutEventsApproximateFailedCount-alarm"
       MetricName: PutEventsApproximateFailedCount
       ComparisonOperator: GreaterThanThreshold
       Namespace: "AWS/Events"
       Statistic: Sum
-      DatapointsToAlarm: 4
-      EvaluationPeriods: 12
-      Dimensions:
-        - Name: EventBusName
```

Comment:
Let's discuss the removal of this, I'm not convinced we should remove it as I can see the filter in the console

Reply:
Hmm I'll have another look but the actual metrics do not seem to have the EventBusName set as a dimension.

![image](https://github.com/user-attachments/assets/9593b2bf-f75b-4545-8df0-7b7c7a9c7faa)


You can see the PutEventsApproximateCallCount metric also does not.

![image](https://github.com/user-attachments/assets/7b7d0b00-2be0-4b7c-bc5e-1296617baa86)

Reply:
@mikebeeby have a lool at https://aws.amazon.com/blogs/compute/enhanced-amazon-cloudwatch-metrics-for-amazon-eventbridge/

Reply:
Ahh nice link, thanks better than anything I have found googling this morning! 
So yeah no dimensions on the PutEventsApproximateFailedCount metric.